assets/js/components/setup-module.js

```diff
@@ -144,8 +144,10 @@ class SetupModule extends Component {
 						>
 							{
 								! blockedByParentModule ?
+									/* translators: %s: module name */
 									sprintf( __( 'Set up %s', 'google-site-kit' ), name ) :
-									sprintf( __( 'Setup Analytics to gain access to %s', 'google-site-kit' ), name )
+									/* translators: %s: module name */
+									sprintf( __( 'Set up Analytics to gain access to %s', 'google-site-kit' ), name )
```

Comment:
Changed from "Setup Analytics" to "Set up Analytics" for consistency, but also because it is the correct form to use here. This caused VRT to fail, hence updates for those as well.